Moonscraper Chart Editor/Assets/Scripts/Game/Charts/IO/Chart/ChartIOHelper.cs

```diff
@@ -63,6 +76,18 @@ public const string
         { c_proDrumsOffset + 3, Note.Flags.ProDrums_Cymbal },       // Blue save num from c_drumNoteNumLookup
         { c_proDrumsOffset + 4, Note.Flags.ProDrums_Cymbal },       // Orange (Green in 4-lane) save num from c_drumNoteNumLookup
         { c_instrumentPlusOffset, Note.Flags.InstrumentPlus },      // Double Kick
+
+        { c_drumsAccentOffset + 1, Note.Flags.ProDrums_Accent },    // Red accent
+        { c_drumsAccentOffset + 2, Note.Flags.ProDrums_Accent },    // Yellow accent
+        { c_drumsAccentOffset + 3, Note.Flags.ProDrums_Accent },    // Blue accent
+        { c_drumsAccentOffset + 4, Note.Flags.ProDrums_Accent },    // Orange accent
+        { c_drumsAccentOffset + 5, Note.Flags.ProDrums_Accent },    // Green accent
+
+        { c_drumsGhostOffset + 1, Note.Flags.ProDrums_Ghost },      // Red ghost
+        { c_drumsGhostOffset + 2, Note.Flags.ProDrums_Ghost },      // Yellow ghost
+        { c_drumsGhostOffset + 3, Note.Flags.ProDrums_Ghost },      // Blue ghost
+        { c_drumsGhostOffset + 4, Note.Flags.ProDrums_Ghost },      // Orange ghost
+        { c_drumsGhostOffset + 5, Note.Flags.ProDrums_Ghost },      // Green ghost
     };
 
         // Default flags, mark as cymbal for pro drums automatically. Also used for choosing whether to write flag information or not if it's like this by default in the first place.
```

Comment:
Well, it's not my comment :P
What should it say? Or should it be removed?

Reply:
Hm, hold on. GitHub is showing a different section than VS Code was.

Reply:
Yeah, I was one off, github can be a bit finiky sometimes. :P

Reply:
I'm still not sure what to do about this comment. Ultimately, this review point shouldn't be relevant to this PR anyways since it was never my comment to begin with, and I don't think I changed anything relevant to it.

Reply:
Yeah, I get that it's got nothing to do with your code but it's just something I noticed is wrong and I want to get it fixed before it gets forgotten about, that's all.